presto-main/etc/catalog/prometheus.properties

```diff
@@ -2,4 +2,5 @@ connector.name=prometheus
 prometheus-uri=http://localhost:9090
 query-chunk-size-duration=1d
 max-query-range-duration=21d
-cache-duration=30s
\ No newline at end of file
+cache-duration=30s
+bearer-token-file=/tmp/bearer_token.txt
```

Comment:
It's unclear to me how this file is used, is it just examples? I'm not sure we should set bearer-token-file to anything by default since Prometheus doesn't have auth normally.

Reply:
The idea was leave it out of the property file and you're not doing bearer auth.
If it is present it's used in: `PrometheusClient::getBearerAuthInfoFromFile()`
Maybe we should include it, but commented out, as the example it was meant to be?

Reply:
I think these files are supposed to be the minimal options you need to set. If i look at the `presto-main/etc/catalog/hive.properties` it's only:

```
#
# WARNING
# ^^^^^^^
# This configuration file is for development only and should NOT be used
# in production. For example configuration, see the Presto documentation.
#

connector.name=hive-hadoop2
hive.metastore.uri=thrift://localhost:9083
```

Which makes me think these configs should just be minimal examples that would allow using these connectors during for development.

Reply:
Changed to match up with other connector configs in cf4bf4a